coreneuron/mechanism/capac.cpp

```diff
@@ -65,87 +65,72 @@ void nrn_mul_capacity(NrnThread*, Memb_list*, int);
 #define nparm 2
 
 void capacitance_reg(void) {
-    int mechtype;
     /* all methods deal with capacitance in special ways */
     register_mech(mechanism, nrn_alloc_capacitance, (mod_f_t)0, (mod_f_t)0, (mod_f_t)0,
                   (mod_f_t)nrn_init_capacitance, -1, 1);
-    mechtype = nrn_get_mechtype(mechanism[1]);
+    int mechtype = nrn_get_mechtype(mechanism[1]);
     _nrn_layout_reg(mechtype, LAYOUT);
     hoc_register_prop_size(mechtype, nparm, 0);
 }
 
-#define cm vdata[0 * _STRIDE]
-#define i_cap vdata[1 * _STRIDE]
```

Comment:
I understand why we should remove this but at the same time it helps lot in reading below code :

```
- i_cap = cfac * cm * _vec_rhs[ni[_iml]];
+ vdata[1 * _STRIDE] = cfac * vdata[0 * _STRIDE] * _vec_rhs[ni[_iml]];
```

Reply:
ok, I can parametrized macro and capitalize them.

Should be more readable.

Reply:
This is derived from mod2c and I wouldn't bother with these auto generated files atm. With NMODL they will get replaced.

Reply:
It was not clear that we want in the future to generate those files again. In this case I would not touch them at all.